backend/test/resolver.spec.js

```diff
@@ -0,0 +1,91 @@
+const { gql } = require('apollo-server-express');
+const { createTestClient } = require('apollo-server-testing');
+const { getTestApolloServer } = require('../server.js');
+const jwt = require('jsonwebtoken');
+const server = getTestApolloServer();
+const initClient = createTestClient(server);
+const query = initClient.query;
+const mutate = initClient.mutate;
+
+// could not outsource these queries/mutations...
+const ALL_ITEMS_QUERY = gql`
+    query itemsQuery {
+        items {
+            id,
```

Comment:
The commas are obsolete, aren't they?

Use the *prettify* button of graphql-playground.